plugins/com.google.cloud.tools.eclipse.login/src/com/google/cloud/tools/eclipse/login/ui/GoogleLoginCommandHandler.java

```diff
@@ -14,20 +14,19 @@
  * limitations under the License.
  */
 
-package com.google.cloud.tools.eclipse.login;
+package com.google.cloud.tools.eclipse.login.ui;
 
-import com.google.cloud.tools.eclipse.login.ui.AccountsPanel;
+import com.google.cloud.tools.eclipse.login.IGoogleLoginService;
+import com.google.cloud.tools.eclipse.login.Messages;
```

Comment:
Hmm... all of our bundles have `Messages` at the top level, so I think I'll leave it there.